Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -183,6 +183,20 @@ public void Delete(string id)
                     }
                 }
 
+                if (!string.IsNullOrEmpty(deploymentInfo.FixedDeploymentId))
+                {
+                    changeSet = new ChangeSet(
```

Comment:
Should we not create the original `ChangeSet` with the appropriate Deployment ID instead of a GUID?

Reply:
(let's continue this conversation on the kudulite thread)